storage/innobase/handler/ha_innodb.cc

```diff
@@ -963,6 +963,8 @@ static SHOW_VAR innodb_status_variables[]= {
   (char*) &export_vars.innodb_buffer_pool_load_status,	  SHOW_CHAR},
   {"buffer_pool_resize_status",
   (char*) &export_vars.innodb_buffer_pool_resize_status,  SHOW_CHAR},
+  {"buffer_pool_load_incomplete",
+  (char*) &export_vars.innodb_buffer_pool_load_incomplete, SHOW_LONG},
```

Comment:
I think that it should be `SHOW_BOOL`. Can we use a compatible type, so that the cast can be avoided?